internal/pkg/initialize/workload.go

```diff
@@ -226,8 +223,8 @@ func (w *WorkloadInitializer) initService(props *ServiceProps) (string, error) {
 	log.Successf(manifestMsgFmt, svcWlType, color.HighlightUserInput(props.Name), color.HighlightResource(path))
 
 	helpText := "Your manifest contains configurations like your container size and port."
-	if props.Port != 0 {
-		helpText = fmt.Sprintf("Your manifest contains configurations like your container size and port (:%d).", props.Port)
+	if len(props.Ports) > 0 {
+		helpText = fmt.Sprintf("Your manifest contains configurations like your container size and %s (:%s).", english.PluralWord(len(props.Ports), "port", "ports"), strings.Trim(strings.Replace(fmt.Sprint(props.Ports), " ", ",", -1), "[]"))
```

Comment:
nit: this is fine but I found it relies on how `fmt.Sprint` format a list of uint. An alternative way that. I had in mind would be
```suggestion
		helpText = fmt.Sprintf("Your manifest contains configurations like your container size and %s (:%s).", english.PluralWord(len(props.Ports), "port", "ports"), strings.Join(applyAll(props.Ports, strconv. FormatUint), ","))
```

https://github.com/aws/copilot-cli/blob/50e492715d85affe75c66c9012a4afb4d242d616/internal/pkg/cli/cli.go#L149

But we would prolly need to change the function signature of `applyAll` to return a different type than `T`